### ISSUE TITLE
Add a key mappings file / make key mappings configurable

### ISSUE THREAD

**Issue by TuckerDowns** [SabakiHQ/Sabaki#455]
This is a feature request for making key-mappings more easily configureable. On my MBP using the function keys for everything is quite a bit slower than having other key maps, such as maybe `cmd+g` for genmove when playing w/ an engine. 

This suggestion could be as simple as just just adding some additional default key-mappings if adding configuration is too large of a request for one dev cycle. 

One common web UX is for `cmd/shift/alt+?` to pull up a modal with the key maps. (`shift+?` in GitHub)

**Comment by TuckerDowns** [comment]
Lots of electron programs have shortcuts that are not handled by the OS. Not sure how that makes this out of scope.

**Comment by yishn** [comment]
Well, lots of Electron programs are sponsored and developed by big tech firms, this isn't. Not sure how you don't get why this is out of scope.

**Comment by TuckerDowns** [comment]
That's true. There might not be developer (your) bandwidth available to implement an idea like this but does that make it out of scope? 

Calling this out of scope suggests to me that if someone else ever developed this feature and submitted it as a PR that you would refuse the PR. At least, that's what out of scope means in my organization "We will not ever support this, even if we had the time or someone did it for free." We have tons of issues in our projects that are marked enhancement and low priority that we just aren't going to get around to.

Sabaki is by far the most beautiful Go interface out there. That doesn't come without a lot of work and so thank you for your efforts. 

I'm not trying to be abrasive here. I'm trying to understand what you mean by "out of scope," which, to me means that this feature will never be accepted into the main project under any circumstance. 

If someone requested a feature for playing cribbage in addition to go that would seem out of scope to me.

**Comment by yishn** [comment]
To clarify, yes, I don't have the bandwidth to implement this, and anyone who wants to do this will have a hard time and the result will be (probably) not up to (my) standards. Even if someone implemented this feature satisfactorily, the result would only be customizable shortcuts, not really worth the huge effort or the bloat that comes with it. 

I don't even think there are many applications (even non-Electron ones) lets their users customize shortcuts. The only ones come into my mind are Microsoft Office and IDEs.

Of course I can't say for sure that I'll never accept a PR which concerns this issue, and I don't want to.  It's not the open source spirit.
